packages/api-client/src/host-client.ts

```diff
@@ -228,7 +228,10 @@ export class HostClient implements ClientProvider {
         return this.client.getStream(`topic/${topic}`, requestInit, { type: contentType });
     }
 
-    async createTopic(id: string, contentType: string): Promise<{ topicName: string }> {
+    async createTopic(id: string, contentType: string, cli? : boolean): Promise<{ topicName: string }> {
+        if (cli) {
+            return this.client.post("topics", { id, "content-type": contentType, cli }, undefined, { json: true, parse: "json" });
+        }
```

Comment:
This can't be done like this. 
Host client is a library which doesn't and can't know about CLI.